### DISCUSSION TITLE
Can I make all of base plugin on top?

### DISCUSSION THREAD

**Discussion by ghost** [lowlighter/metrics#725] [Q&A] [answered]
How do I order base plugin (incule header, activity, community, repositories) to align (top) and there are other plugins with `config_order: base ... , isocalendar, languages, ...` ?

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

If you're talking about changing the default plugin order, it is indeed with `config_order` option (with the default being [this](https://github.com/lowlighter/metrics/blob/master/source/templates/classic/partials/_.json) for the classic template). So if you wanted to order `base.repositories` before `base.header`, you'd use `config_order: base.repositories, base.header` for example. Unspecified plugins will just use the default order.

All plugins should support this feature.

Some plugins which have a `plugin_*_sections` option may also change the order of the section displayed within the plugin.

If you need more customization, I'd advise to fork this repository and directly edit [partials](https://github.com/lowlighter/metrics/tree/master/source/templates/classic/partials) and use it in your workflow instead
